PrintContentOfTextFileToConsole/src/Main.java

```diff
@@ -0,0 +1,20 @@
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Collection;
+
+
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+        File file = new File("data.txt");
```

Comment:
What if I want to print a file with another name?  It should come as a a value via `args` array.